subprojects/logging/src/main/java/org/gradle/internal/logging/console/WorkInProgressRenderer.java

```diff
@@ -170,7 +170,7 @@ private void addDirectChildOperationId(OperationIdentifier parentId, OperationId
     private void removeDirectChildOperationId(OperationIdentifier parentId, OperationIdentifier childId) {
         Set<OperationIdentifier> children = parentIdToChildrenIds.get(parentId);
         if (children == null) {
-            throw new IllegalStateException("");
+            return;
```

Comment:
Good call. I should probably even write a test for this. 

This had to change because `DefaultBuildOperationExecutor` spawns a _lot_ more progress loggers and we do not want to attach all operations to labels anymore. Therefore some operations like a phase completing would not have any children to remove because it intentionally wasn't attached.